l10n_it_pos_fatturapa/i18n/it.po

```diff
@@ -0,0 +1,86 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* l10n_it_pos_fatturapa
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-12-02 17:22+0000\n"
+"PO-Revision-Date: 2019-12-02 17:22+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_it_pos_fatturapa
+#. openerp-web
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:48
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:102
+#, python-format
+msgid "Code"
+msgstr "Codice"
+
+#. module: l10n_it_pos_fatturapa
+#. openerp-web
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:8
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:70
+#, python-format
+msgid "Company"
+msgstr ""
+
+#. module: l10n_it_pos_fatturapa
+#. openerp-web
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:54
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:108
+#, python-format
+msgid "EORI"
+msgstr ""
+
+#. module: l10n_it_pos_fatturapa
+#. openerp-web
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:20
+#: code:addons/l10n_it_pos_fatturapa/static/src/xml/pos.xml:80
+#, python-format
+msgid "First Name"
```

Comment:
Dimenticavo.
Se vuoi eliminare questo fastidioso problema dell'etichetta su due righe

![immagine](https://user-images.githubusercontent.com/227940/71558798-30f36400-2a57-11ea-8be0-c2436e4a022c.png)

al posto di "First Name" e "Last Name" potresti usare "Name" e "Surname" .

Reply:
Giusto, però sarebbe da fare nel modulo `pos_partner_firstname`

Reply:
> Giusto, però sarebbe da fare nel modulo `pos_partner_firstname`

Corretto, vedo di aprire una PR allora.

Reply:
> Giusto, però sarebbe da fare nel modulo `pos_partner_firstname`

Fatto in https://github.com/OCA/pos/pull/433